manifests/g/GnuWin32/Tree/1.5.2.2/GnuWin32.Tree.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: GnuWin32.Tree
+PackageVersion: 1.5.2.2
+PackageLocale: en-US
+Publisher: GnuWin32 <gnuwin32.sourceforge.net>

```

```suggestion
Publisher: GnuWin32
```
This needs to match Control Panel.